framework/wazuh/engine/request_message.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from typing import Dict, Any, Optional
+from wazuh.engine.commands import EngineCommand
+
+
+class EngineRequestMessage:
+    """ Represents a request to the engine"""
+    def __init__(self, version: int):
+        """
+        Initialize the EngineRequestBuilder object.
+
+        Parameters
+        ----------
+        version : str
+            The version of the engine request.
+        """
+        self.internal_dict: Dict[str, Any] = {"version": version}
```

Comment:
This variable could be private

Reply:
Changed to private in commit 244ec427c775389734b5ac15edac1f4e0a444612
https://github.com/wazuh/wazuh/blob/244ec427c775389734b5ac15edac1f4e0a444612/framework/wazuh/core/engine/request_message.py#L9-L20